### ISSUE TITLE
'Object has been destroyed' in plugin-electron-app-breadcrumbs/app-breadcrumbs.js

### ISSUE THREAD

**Issue by jeremyspiegel** [bugsnag/bugsnag-js#1716]
### Describe the bug
The Electron plugin app-breadcrumbs.js uses `debounce` for tracking the `BrowserWindow.move` event: https://github.com/bugsnag/bugsnag-js/blob/next/packages/plugin-electron-app-breadcrumbs/app-breadcrumbs.js#L120

I'm seeing lots of bugsnag errors `Object has been destroyed` pointing to the use of the `BrowserWindow` in that event handler after the `BrowserWindow` has been destroyed:
```
TypeError Object has been destroyed 
    node_modules/@bugsnag/plugin-electron-app-breadcrumbs/app-breadcrumbs.js:121:39 BrowserWindow.<anonymous>
    node_modules/lodash.debounce/index.js:160:19 invokeFunc
    node_modules/lodash.debounce/index.js:207:14 trailingEdge
    node_modules/lodash.debounce/index.js:195:14 Timeout.timerExpired [as _onTimeout]
    node:internal/timers:557:17 listOnTimeout
    node:internal/timers:500:7 processTimers
```

We might want to clear the `browserWindow` in the `closed` event and not try to use it later in the `moved` event.

### Environment
* Bugsnag version: "@bugsnag/electron": "^7.16.2",
* Browser framework version (if any):
    * Vue:
    * React:
    * Angular:
* Server framework version (if any):
    * Koa:
    * Express:
    * Restify:
* Browser version (e.g. chrome, safari): Electron
* Device (e.g. iphonex): macOS

**Comment by luke-belton** [comment]
Hi @jeremyspiegel - thanks for raising this, our engineering team are going to investigate a fix for this issue. We'll keep you updated on this thread 👍

**Comment by luke-belton** [comment]
Hi @jeremyspiegel - [a fix](https://github.com/bugsnag/bugsnag-js/pull/1722) for this was released in [`v7.16.4`](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.16.4) of `bugsnag-js` 🎉  thanks again for reporting this!

**Comment by jeremyspiegel** [comment]
Thanks for the fix!
